packages/react-instantsearch-dom-maps/src/__tests__/GoogleMaps.js

```diff
@@ -522,13 +516,13 @@ describe('GoogleMaps', () => {
       simulateMapReadyEvent(google);
       simulateEvent(mapInstance, 'center_changed');
 
-      expect(wrapper).toMatchSnapshot();
+      expect(wrapper.html()).toMatchSnapshot();
```

Comment:
That's the easiest solution I found not to change all the tests.
Otherwise, we can't see the generated tree because of the first component being the `Provider`.